upgrade-2.x.sh

```diff
@@ -290,6 +290,21 @@ if ! version_gt "$VERSION" "$preferred_hostos_version" &&
     esac
 fi
 
```

Comment:
Should that version be 2.0.6 in the commit message?

Reply:
Yes, fixed!